### ISSUE TITLE
Power *not* working?

### ISSUE THREAD

**Issue by VolumeZero** [carmineos/fivem-handling-editor#12]
When applying a preset with finitialdriveforce it doesn't work until I go into vMenu and toggle the engine upgrade. Can be any level of engine. 
Ive reinstalled this on a server with no other resources and still the same issue. 
Ive also been on servers where I can apply my preset without this issue.

**Comment by carmineos** [comment]
This seems one of the things that should be handled by the fivem API themselves, the extra-natives to edit handling might require a check and/or more documentation

**Comment by Pepper94** [comment]
> This seems one of the things that should be handled by the fivem API themselves, the extra-natives to edit handling might require a check and/or more documentation

No, it's from your handling editor script. FiveM api has nothing to do with this, I made my own handling editor and it works without any issues. Your handling editor is very good overally, but it has some bugs you still haven't bothered to fix after a whole year. Not only finitialdriveforce does not apply, but fInitialDriveMaxFlatVel as well. You must use "SetVehicleEnginePowerMultiplier" after fInitialDriveMaxFlatVel and "SetVehicleMod" after fInitialDriveForce for those values to apply in-game.

P.S. I enjoy your work a lot, keep creating!
